main.go

```diff
@@ -204,6 +208,14 @@ func validate(in configInput) error {
 		return fmt.Errorf("config-mesh may only contain letters, numbers, and '-'. It must begin with a letter and must not exceed 64 characters in length")
 	}
 
+	// If in.configMesh != "", then do not validate for generateMeshId constraints,
+	// since the value configMesh would be used as MeshID in generate()
+	if in.configMesh == "" && in.generateMeshId {
```

Comment:
Nit: Maybe make this an `else if` hanging off of the `if in.generateMeshId` below?